File: app/src/main/java/ru/prsolution/winstrike/mvp/models/UserProfileObservable.java
package ru.prsolution.winstrike.mvp.models;

import android.databinding.BaseObservable;
import android.databinding.Bindable;

import ru.prsolution.winstrike.BR;

public class UserProfileObservable extends BaseObservable {
    private String name;

    @Bindable
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
        notifyPropertyChanged(BR.name);
    }
}

File: app/src/main/kotlin/ru/prsolution/winstrike/domain/models/UserProfileObservable.kt
package ru.prsolution.winstrike.domain.models


import androidx.databinding.BaseObservable
import androidx.databinding.Bindable

class UserProfileObservable : BaseObservable() {
	@get:Bindable
	//        notifyPropertyChanged(name);
	var name: String? = null
}
